feat(ins): add resize doc (#4972)
scaleway/docs-content: pages/instances/how-to/resize-instances.mdx

---
meta:
  title: How to resize Instances
  description: Learn how to adjust the size of your Instances from the Scaleway console to handle varying workloads effectively.
content:
  h1: How to resize Instances
  paragraph: Learn how to adjust the size of your Instances from the Scaleway console to handle varying workloads effectively.
tags: instance scaling vertically compute
dates:
  validation: 2025-05-15
  posted: 2025-05-15
categories:
  - compute
---

You can adjust the size of your Scaleway Instances (also known as vertical scaling) to meet changing workload demands and optimize resource usage.
Whether you need to scale up to handle increased traffic, scale down to reduce costs, or simply reconfigure your Instance to better align with your application’s requirements, vertical scaling lets you increase or decrease computational power without the hassle of destroying and recreating Instances.

> Scaleway’s flexible infrastructure allows you to modify CPU and RAM allocation, as well managing your Block Storage vomumes directly from the console, ensuring your resources always match your needs.

Reviewer: ```suggestion
Scaleway’s flexible infrastructure allows you to modify CPU and RAM allocation, as well as manage your Block Storage volumes directly from the console, ensuring your resources always match your needs.
```

> Scaleway’s flexible infrastructure allows you to modify CPU and RAM allocation, as well managing your Block Storage vomumes directly from the console, ensuring your resources always match your needs.

Reviewer 2: ```suggestion
Scaleway’s flexible infrastructure allows you to modify CPU and RAM allocation, as well managing your Block Storage volumes directly from the console, ensuring your resources always match your needs.
```

> Alternatively, you can also resize your Instance using the command-line by following the guide [Changing the commercial type of an Instance with the API or CLI](/instances/api-cli/migrating-instances/).

Reviewer 2: ```suggestion
Alternatively, you can also resize your Instance using the command line by following the guide [Changing the commercial type of an Instance with the API or CLI](/instances/api-cli/migrating-instances/).
```
